akka-typed/src/main/scala/akka/typed/internal/ActorSystemImpl.scala

```diff
@@ -63,13 +64,13 @@ Distributed Data:
  */
 
 private[typed] class ActorSystemImpl[-T](
-  override val name:       String,
-  _config:                 Config,
-  _cl:                     ClassLoader,
-  _ec:                     Option[ExecutionContext],
-  _userGuardianBehavior:   Behavior[T],
+  override val name: String,
+  _config: Config,
+  _cl: ClassLoader,
+  _ec: Option[ExecutionContext],
```

Comment:
I might have forgotten to reformat after the last Eclipse save—it is a pain to sync these settings.